src/GitHub.UI.Reactive/Controls/Validation/ValidationMessage.cs

```diff
@@ -33,7 +34,7 @@ public ValidationMessage()
                 .Do(CreateBinding)
                 .Select(control =>
                     Observable.Merge(
-                        this.WhenAnyValue(x => x.ShowError),
+                        this.WhenAnyValue(x => x.ShowError).Where(x => userHasInteracted),
```

Comment:
So I'm curious, why is ShowError signaled here has having been changed if on line 28 it's only set when the ValidationResult is not null? Is `WhenAnyValue` always triggered immediately on setup, and is there a way of having it not triggered until something actually changes in the property?